content/en/docs/concepts/security/controlling-access.md

```diff
@@ -136,34 +136,6 @@ The cluster audits the activities generated by users, by applications that use t
 
 For more information, see [Auditing](/docs/tasks/debug/debug-cluster/audit/).
 
-## API server ports and IPs
-
-The previous discussion applies to requests sent to the secure port of the API server
-(the typical case).  The API server can actually serve on 2 ports:
-
-By default, the Kubernetes API server serves HTTP on 2 ports:
-
-  1. `localhost` port:
-
-      - is intended for testing and bootstrap, and for other components of the master node
-        (scheduler, controller-manager) to talk to the API
-      - no TLS
-      - default is port 8080
-      - default IP is localhost, change with `--insecure-bind-address` flag.
-      - request **bypasses** authentication and authorization modules.
-      - request handled by admission control module(s).
-      - protected by need to have host access
-
-  2. “Secure port”:
-
-      - use whenever possible
-      - uses TLS.  Set cert with `--tls-cert-file` and key with `--tls-private-key-file` flag.
-      - default is port 6443, change with `--secure-port` flag.
-      - default IP is first non-localhost network interface, change with `--bind-address` flag.
-      - request handled by authentication and authorization modules.
-      - request handled by admission control module(s).
-      - authentication and authorization modules run.
-
```

Comment:
Instead of removing the whole section, I'd suggest we keep the text about the "secure port" and rewrite it.

Reply:
What do you think about still removing this section, but integrating the configuration flags, default port and default listening IP into the first section "Transport security"? There really is not much other new information in this section, and most of the bullet points are kind of obsolete as they mainly are there for distinction between the insecure port.